packages/fxa-content-server/app/scripts/templates/ready.mustache

```diff
@@ -30,7 +30,9 @@
           {{/emailVerified}}
 
           {{^emailVerified}}
-              <p class="account-ready-generic">{{#t}}Your account is ready!{{/t}}</p>
+            {{#noRelierServiceDefault}}
+              <p class="account-ready-generic">{{#t}}You are now ready to use %(noRelierServiceDefault)s{{/t}}</p>
+            {{/noRelierServiceDefault}}
```

Comment:
I pinged Ryan about this change in Slack and he gave it the thumbs up.

TBH we probably do not need to check for `noRelierServiceDefault` if every one of these views is using the `service-mixin.js` We could just assume in our mustache files that `noRelierServiceDefault` is going to be truthy if `serviceName` is falsey but I'm not sure that's obvious unless you pull up the mixin. 🤷‍♀️ I can easily pull the checks out if that's preferred, though.